### PR TITLE
refactored Login from a class to a hook

### PR DESCRIPTION
**Pull request by dylvaz**
refactored Login per #12

### CODE REVIEW

**Comment by mikhael28** [comment]
Hey Dylan, this looks great - I am quickly wrapping up some work on the `CreateUser` container, to verify that people can 'sign' up without my needing to use the Admin portal. Once I take a look at that, I will review your PR - from my initial glance, everything checks out and doesn't require additional review - I just need to test it to make sure it works before pushing it to prod. After this, I will go ahead and set up a deployment environment for the 'develop' branch, where we I can quickly set up another CI deployment process. The `main` branch is already set up for CI, just not with GitHub actions.

In the meantime, I would love to learn more about you and what your goals are - would you mind introducing yourself here? https://github.com/mikhael28/paretOS/discussions/17

File: src/profile/Login.js [hunk 1]
```diff
@@ -13,88 +13,88 @@ import logo from "../assets/Pareto_Lockup-01.png";
...
+  const validateForm = () => {
+    return values.email.length > 0 && values.password.length > 0;
+  };
 
-  handleChange = (event) => {
-    this.setState({
-      [event.target.id]: event.target.value,
-    });
+  const handleChange = (event) => {
+    setValues({ ...values, [event.target.id]: [event.target.value] });
```
Comment (mikhael28) [inline src/profile/Login.js:34]: This is mostly right, but you do not need a bracket around the event.target.value. If you go to the handleSubmit, and you console.log the values - you will find that the string is inside of an array, which triggers an Error from Amplify's text validation. 

Please remove the brackets in another commit, and push it up. After that, I will accept the PR - it's working outside of that. Then, go ahead and make an account for yourself after pulling the new changes from the upstream `main` branch. I have re-enabled self-service registration!

**Review by mikhael28 — CHANGES_REQUESTED** [review]
Only one comment, please make that one change and I will happily accept the PR! Should take 30 seconds.

File: src/profile/Login.js (hunk 1 above)
Comment (dylvaz) [reply src/profile/Login.js]: Thanks for catching that! I've gone ahead and made that change.

**Review by mikhael28 — APPROVED** [review]

**Pull request merged**